uk.co.mrbenshef.TeX-Match.json

```
@@ -0,0 +1,45 @@
+{
+    "app-id": "uk.co.mrbenshef.TeX-Match",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
```

Update runtime to 20.08
```suggestion
    "runtime-version": "20.08",
```